multi-node/aws/pkg/cluster/cluster.go

```diff
@@ -337,3 +352,25 @@ func (c *Cluster) validateDNSConfig(r53 r53Service) error {
 
 	return nil
 }
+
+func stackEventErrMsgs(events []*cloudformation.StackEvent) []string {
+	var errMsgs []string
+
+	for _, event := range events {
+		if aws.StringValue(event.ResourceStatus) == cloudformation.ResourceStatusCreateFailed {
+			// Only show actual failures, not cancelled dependent resources.
+			if aws.StringValue(event.ResourceStatusReason) != "Resource creation cancelled" {
```

Comment:
A failure to create a resource causes cancellation failures for the resources that depend on it, which don't help with debugging. I wanted to only show root causes.

If you or @colhom or @mumoshu think showing the cancellations is worth while, I can remove that check.